dbt/adapters/protocol.py

```diff
@@ -93,81 +93,58 @@ class AdapterProtocol(  # type: ignore[misc]
     ConnectionManager: Type[ConnectionManager_T]
     connections: ConnectionManager_T
 
-    def __init__(self, config: AdapterRequiredConfig) -> None:
```

Comment:
Mind backing out the purely formatting changes?

Reply:
Ahh! This exact issue with `def fn(...): \n ...` and Black removing the `\n`'s happened in another project I maintain, except in reverse (happened in CI, not locally). Very annoying! https://github.com/ewels/rich-click/actions/runs/8243539852/job/22544297678#step:5:281